src/widgets/rating-menu/rating-menu.tsx

```diff
@@ -78,6 +78,12 @@ export type RatingMenuCSSClasses = {
   count: string | string[];
 };
 
+type RatingMenuRendererCSSClasses = Required<
```

Comment:
I'm not sure I fully understand the need of these types

Reply:
This type is used to type `cssClasses` in the renderer by derivating it from `RatingMenuCSSClasses` (the difference being that for each property we only accept `string` and not `string[]`)

PS: the Required is actually not useful here since all properties are already required in `RatingMenuCSSClasses`, I'll fix that.